docs/user/getting_started.md

```diff
@@ -0,0 +1,252 @@
+# Basic Usage
+
+To start running Mizar, first you will need a Kubernetes cluster. Running Mizar will  install experimental features on your cluster. **It is recommended that you use a test cluster to run Mizar.**
+
+## Prerequisites
+
+### Kind (Kubernetes in Docker)
+
+**Note**: As of writing this, there is an error in the installation instructions for Kind.
+The commands below can be used to download and install the latest release binary for Kind.
+
+MacOS/Linux
+```
+ver=$(curl -s https://api.github.com/repos/kubernetes-sigs/kind/releases/latest | grep -oP '"tag_name": "\K(.*)(?=")')
+curl -Lo kind "https://github.com/kubernetes-sigs/kind/releases/download/$ver/kind-$(uname)-amd64"
+chmod +x kind
+mv kind /usr/local/bin
+```
+
+The recommended way to try out Mizar is with Kind.
+Kind can be used to run a multi-node Kubernetes cluster with Docker containers locally on your machine.
+You can find [instructions for installing Kind on it's official site here.](https://kind.sigs.k8s.io/docs/user/quick-start/)
+
+* Enter the mizar directory and run the ```bootstrap.sh``` script.
+    * This script will install the neccessary components to compile Mizar, and run it's unit tests. These include
+        * [Clang-7](https://clang.llvm.org) (For code compilation)
+        * [Llvm-7](https://llvm.org) (For code compilation)
+         * [Cmocka](https://cmocka.org) (For unit testing)
+        * [Build Essentials](https://packages.ubuntu.com/xenial/build-essential) (For code compilation)
+        * [Python](https://www.python.org) (For running the management plane and tests)
+        * [Docker](https://www.docker.com) (For management plane and tests)
+    * There is also an optional kernel update included in the script if you wish to update to a compatible version
+
+### Linux Kernel Update
+
+For TCP to function properly, you will need to update your Kernel version to at least 5.6-rc2 on every node. A script, ```kernel_update.sh``` is provided in the Mizar repo to download and update your machine's kernel if you do not wish to build the kernel source code yourself.
+
+
+## Setup
+
+### Compiling Mizar
```

Comment:
Shall we move the compilation and test running to the developer guide?

Reply:
Sure, I've moved it to the developer guide.